doc: getting_started: Split up the three win guides

SebastianBoe opened the pull request:
Several users have reported confusion about whether one should stop at
the 'Using MSYS2' guide, or continue following the guide.

To resolve this we split up the three guides into separate documents
that need to be selected, this should be familiar to the user as it
looks just like how the three OS guides are split up.

Signed-off-by: Sebastian Bøe <sebastian.boe@nordicsemi.no>

IvanSanchez commented on doc/getting_started/installation_win_wsl.rst:
```diff
@@ -0,0 +1,28 @@
+   Started Guide which can be found here: :ref:`installation_linux`
+
+.. _GNU ARM Embedded: https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads
+.. _Chocolatey website: https://chocolatey.org/
```
These links are used from the two other installation guides. Please double-check and move the links to their respective files.

dbkinder reviewed (changes requested):
There's a much simpler way to address the reported "confusion about whether one should stop at
the 'Using MSYS2' guide, or continue following the guide." problem.  Add a sentence at the end of each setup section that says your Windows environment is set up and ready to use.  We could also change the titles to be "Option 1: Using Windows Command Prompt", "Option 2: Using MSYS2", etc.

SebastianBoe commented:
@dbkinder : I considered that as well, but thought that this would be cleaner, as it follows the same pattern as the _mac, _lin, _win guides. Will defer to your experience and push improved documentation.

SebastianBoe committed:
doc: getting_started: Make it more obvious how to follow the guide

Several users have reported confusion about whether one should stop at
the 'Using MSYS2' guide, or continue following the guide.

To resolve this we title the three options as Option 1,2,3, and change
the wording of the second guide to make it clear that it is an
alternative, not an additional step.

Signed-off-by: Sebastian Bøe <sebastian.boe@nordicsemi.no>

SebastianBoe commented:
> There's a much simpler way to address the reported "confusion about whether one should stop at
> the 'Using MSYS2' guide, or continue following the guide." problem. Add a sentence at the end of each setup section that says your Windows environment is set up and ready to use. We could also change the titles to be "Option 1: Using Windows Command Prompt", "Option 2: Using MSYS2", etc.

@dbkinder : Added the option titles to make it clear that these are mutually exclusive options. And added a sentence at the top of the second option for the same effect. Did not modify the end of the section, because I was unable to find a wording that was better than what was already there:

> This should check that all the tools and toolchain are set up correctly for
> your own Zephyr development.
>

tyrantlizard77 reviewed (approved):
Looks fine.
I do wonder why we maintain 3 different methods. Seems like Options 1 and 3 ought to be enough?

dbkinder commented on doc/getting_started/installation_win.rst:
```diff
@@ -168,11 +168,12 @@ packages from their respective websites.
 your own Zephyr development.
 
-Using MSYS2
+Option 2: MSYS2
```
The title underline must be at least as long as the title, so add some more ``====`` 's

SebastianBoe replied:
Done

dbkinder commented on doc/getting_started/installation_win.rst:
```diff
@@ -339,7 +340,7 @@ To build for the ARM-based Nordic nRF52 Development Kit:
 your own Zephyr development.
 
-Using Windows 10 WSL (Windows Subsystem for Linux)
+Option 3: Windows 10 WSL (Windows Subsystem for Linux)
```
The title underline must be at least as long as the title, so add some more ``====``'s

SebastianBoe replied:
Done

SebastianBoe commented:
@dbkinder : Please re-review, all feedback has been addressed.

dbkinder reviewed (approved):
Thanks!
